tcp/pcb.ml

```diff
@@ -465,6 +468,10 @@ struct
       Lwt.return_unit
     | None ->
       Tx.send_rst t id ~sequence ~ack_number ~syn ~fin
+    | exception ex ->
+      PCB_Log.err (fun f -> f "uncaught exception looking up handler for TCP port %d:@ %s"
+                      id.WIRE.local_port (Printexc.to_string ex));
+      Lwt.return_unit
```

Comment:
Isn't this a public entrypoint defined in mirage-types?

https://github.com/mirage/mirage/blob/76575138f27ccfbff2a073ba997c54d4afb84149/types/V1.mli#L661

(using a stack is optional, but in that case it shouldn't fail indeed)